test/zhr_devs/identity_management/commands_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule ZhrDevs.IdentityManagement.CommandsTest do
+  use ExUnit.Case, async: true
+
+  import Commanded.Assertions.EventAssertions
+
+  alias ZhrDevs.IdentityManagement
+
+  test "dispatch of the Login command emits an event with tighten identity, hashed_identity and login_at stamp" do
+    command =
+      IdentityManagement.Commands.Login.new(
+        identity: "example",
+        hashed_identity: "sPJI3B_c0reGPtFtacqjHWDzSUh_AkkujufycDOYweI="
+      )
+
+    :ok = IdentityManagement.App.dispatch(command)
+
+    assert_receive_event(IdentityManagement.App, IdentityManagement.Events.LoggedIn, fn event ->
```

Comment:
Oh wow, what's `assert_receive_event`? Looks cool.

Reply:
That's the part of the `import Commanded.Assertions.EventAssertions` module provided by Commanded, it includes several similar assertions as well.